docs/guide/exe_verilator.rst

```diff
@@ -780,7 +780,7 @@ Summary:
    be useful in makefiles. See also :vlopt:`-V`, and the various
    :file:`*.mk` files.
 
-   Feature may be one of the following: COROUTINES, SYSTEMC, ASAN.
+   Feature may be one of the following: COROUTINES, SYSTEMC, ASAN, GCOV.
 
```

Comment:
GCOV feels wrong, we might have some other program do coverage or e.g. do MSVC coverage and should be same indication.

Also calling it COVERAGE (or even GCOV) can be confused with using coverage in SystemVerilog.  Maybe SELF-COVERAGE or something (suggest better naming).  Probably should use same --enable-self-coverage wording for config.

Nit: Let's sort this list.

Reply:
Agree, I don't really like this, but I also don't like that developer only stuff is exposed in the docs here. I picked "GCOV" because for example `clang` has both a `gcov` based coverage flow, and a "Source-based Code Coverage" flow of its own (which seems superior, but didn't want to migrate to that right now), so might need to know the difference.

How about `DEV_GCOV` (and also `DEV_ASAN`) and then not include these in the user docs? One should never ship an asan or gcov build.

Then rename to `--enable-dev-gcov` and `--enable-dev-asan`? and change the configure macros `CFG_WITH_DEV_GCOV`? Not sure we want to touch `--enable-ccwarn`

Reply:
Ok, I will do a pre-pull to rename the asan ones.